<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>source_peizhi</class>
 <widget class="QDialog" name="source_peizhi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择数据库模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pattern"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>选择表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_table"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_add_data">
     <property name="text">
      <string>添加数据表</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>输入发布用户</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ip"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_port"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add_user">
        <property name="text">
         <string>添加用户</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_over">
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
